get-started/02_our_app.md

```diff
@@ -121,11 +121,13 @@ Now that you have an image, you can run the application in a [container](../get-
 1. Start your container using the `docker run` command and specify the name of the image you just created:
 
    ```console
-   $ docker run -dp 3000:3000 getting-started
+   $ docker run -dp 127.0.0.1:3000:3000 getting-started
```

Comment:
Maybe out of scope for this PR, but for examples, I have also tried to use different port numbers for the host and container ports. Doing so can make it clearer that it's a mapping, and in some cases helps disambiguate what port is "host" and what port is "container" (e.g. `-p 8080:80`)

Reply:
not a bad idea (especially for port `80` 🙈) I think we should try and do that going forward. But not for this PR.